### DISCUSSION TITLE
help me trackball does not work

### DISCUSSION THREAD

**Discussion by dorongss** [Yowkees/keyball#519] [Q&A]
The trackball doesn't work at all.
I don't know what the hell the problem is. Please solve it.
If there's no solution
Please tell me how I can buy related parts again
![KakaoTalk_20240123_195123730](https://github.com/Yowkees/keyball/assets/157365330/3ff40849-5293-40c7-b380-13e846a7a433)
![KakaoTalk_20240123_195123730_01](https://github.com/Yowkees/keyball/assets/157365330/a5eaf7fa-0dc2-4ded-935f-c12a5124eafc)
![KakaoTalk_20240123_195123730_02](https://github.com/Yowkees/keyball/assets/157365330/3eb8260b-b9fd-448f-9eab-bcbf07d26fd1)

**Comment by koron** [comment]
(converted from issue)

I don't know if my point is correct.
But from this photo, it looks like the solder on the MOSI pin is shorted to the capacitor.

![image](https://github.com/Yowkees/keyball/assets/468368/29b22581-9c5d-476c-ab91-c6c7672a3cdb)
